### DISCUSSION TITLE
Enterprise Deployment on Windows Server 2022

### DISCUSSION THREAD

**Discussion by y4i4vkadc3** [2fast-team/2fast#139] [General]
Is it possible to use this app on Windows Server 2022?
Is there any information how it works?
Thank you.

**Comment by jp-weber** [comment]
Yes it is possible. As unfortunately the Store is not available and to my knowledge MS Intune is not supported either, you can install the application manually, [see releases](https://github.com/2fast-team/2fast/releases), or you can [install Chocolatey](https://community.chocolatey.org/packages/2fast), for example. 

I have not yet got [winget](https://github.com/microsoft/winget-cli) to run on the server, but this is only experimental for Server 2022 and not really supported: 
> [!NOTE]
The client requires Windows 10 1809 (build 17763) or later at this time. Windows Server 2019 is not supported as the Microsoft Store is not available nor are updated dependencies. It may be possible to install on Windows Server 2022, this should be considered experimental (not supported), and requires dependencies to be manually installed as well.

**Comment by y4i4vkadc3** [comment]
Thank you for answer.
We have addressed our requirements in a different way. So i will suspend this discussion from my side.
